src/components/node-list/node-list-items.test.js

```diff
@@ -24,7 +24,7 @@ const ungroupNodes = groupedNodes =>
 describe('node-list-selectors', () => {
   describe('getFilteredNodes', () => {
     const nodes = getGroupedNodes(mockState.animals);
-    let searchValue = 'bear';
+    let searchValue = 'Bear';
```

Comment:
It is currently capitalised to match the format of the actual data returned - currently `node.name` for `data` type nodes is formatted to start in caps ( however this only applies to `data` and `parameter` type nodes and not for `task` type nodes - don't ask me why, I have asked Kiyo about this and tried to modify the `animals` data file instead but this file has to match the actual data that was returned from the BE). 

This is one very good example to show the small discrepancies of the expectations of the dataset format - it might br worth to also raise the question to the BE team with regards to this inconsistent casing of the node names.